src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java

```diff
@@ -105,14 +130,25 @@ public final void moveWrist(Rotation2d angle) {
     }
 
     // Enable feedback control
-    public abstract void setFeedbackEnabled(boolean enabled);
+    public final void setVoltageControl(Voltages voltages) {
+        trajectory = Optional.empty();
+        this.voltages = Optional.of(voltages);
+    }
+
+    public final void enableVoltageControl() {
+        setVoltageControl(new Voltages(0, 0));
+    }
+
+    public final void disableVoltageControl() {
+        this.voltages = Optional.empty();
```

Comment:
This should set a setpoint for the encoders after you turn off the voltage because it might go to a bad setpoint.

Reply:
-by Sam (not me)